examples/01_brain_alignment/plot_2_aligning_brain_sparse.py

```diff
@@ -694,7 +694,7 @@ def plot_surface_map(
 # plus ``(1 - id_reg)`` times the transport plan computed by FUGW.
 
 # Set of weights for the regularization parameter
-id_reg = [.0, .5, 1.]
+id_reg = [1.0, 0.5, 0.0]
```

Comment:
Is there a way to assess that there is no arbitrariness between what is shown and what is asserted ?

Reply:
Sure thing, I added an enumeration and a sort. The plot is not relevant on the order of ``id_reg`` anymore.